rpc_util.go

```diff
@@ -172,16 +168,27 @@ func encode(c Codec, msg interface{}, pf payloadFormat) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
-		length = uint(len(b))
```

Comment:
This is intentionally. Please keep uint (note that len(...) returns int). uint32 makes the overflow checking in line#177 void.